src/win/win-helpers.cpp

```diff
@@ -491,14 +491,15 @@ namespace librealsense
                 }
 
                 // Enumerate all imaging devices
-                for (int member_index = 0; ; ++member_index)
+                for (DWORD member_index = 0; member_index < 0x10000; ++member_index)
```

Comment:
This is only some limit to avoid infinite loop. No real meaning for this number - only assuming that there will be no system with 0x10000 (65,536) devices.
I ll add a comment.